src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -700,6 +705,12 @@ namespace PurchasedAirManager {
 
 			SetupOutputVariable( "Zone Ideal Loads Hybrid Ventilation Available Status []", PurchAir( PurchAirNum ).AvailStatus, "System", "Average", PurchAir( PurchAirNum ).Name );
 
+			//air flows
```

Comment:
@JasonGlazer Looks like the indentation is wrong around here.  Maybe spaces instead of tabs?